platform/core.startup/src/org/netbeans/core/startup/Splash.java

```diff
@@ -544,7 +544,7 @@ private void adjustText(String text) {
             if (text == null)
                 return ;
 
-            if (statusBox.fm == null)
+            if (statusBox == null ||statusBox.fm == null)
```

Comment:
Please add a whitespace after `||`. Maybe, should use curly braces(`{}`).

Reply:
Added the missing whitespace and braces around if statements.